src/api/spec/factories/project.rb

```diff
@@ -102,6 +102,7 @@
           create(:build_flag, status: 'disable', project: evaluator.target_project)
           create(:publish_flag, status: 'disable', project: evaluator.target_project)
           update_project.projects_linking_to << evaluator.target_project
+          update_project.store
```

Comment:
Too late but

```
if CONFIG['global_write_through']
  update_project.store
end
```
would be better, right? Otherwise I assume this will fail if the backend is not running, right?